src/fluidplayer.js

```diff
@@ -35,7 +35,7 @@ const FP_RUNTIME_DEBUG = typeof FP_DEBUG !== 'undefined' && FP_DEBUG === true;
 
 let playerInstances = 0;
 
-const fluidPlayerClass = function () {
+const fluidPlayerClass = function() {
```

Comment:
There's been an auto reformat done with something like prettier I see, that's a good idea but not in the scope of this ticket. We want to do this in the future, but it has to be defined in a ticket how we want to do this. It's just how the processes work in the company. For now you can revert the formatting.